core-bundle/src/EventListener/DataContainer/RecordPreviewListener.php

```diff
@@ -60,6 +60,10 @@ public function storePrecompiledRecordPreview(DataContainer $dc, string $undoId)
                 ->fetchAssociative()
             ;
 
+            if (!$row) {
+                return;
+            }
```

Comment:
Should we move this and the database query above the `try` block?

Reply:
Seems possible yes. Other exceptions captured here might happen again when we try to update the database anyway.

Reply:
I updated accordingly